admin/src/js/controllers/display-privacy-policies-preview.js

```diff
@@ -5,14 +5,20 @@ angular
     $sanitize,
     $scope,
     $timeout,
-    $uibModalInstance,
-    PrivacyPolicies
+    $uibModalInstance
   ) {
     'use strict';
     'ngInject';
 
     const getHtml = string => {
-      return $sanitize(PrivacyPolicies.decodeUnicode(string));
```

Comment:
This is a refactor that I found while working out the privacy policy fields in user settings - by inlining this one function we can drop the entire service.